browser/main/lib/dataApi/attachmentManagement.js

```diff
@@ -386,6 +386,17 @@ function removeStorageAndNoteReferences (input, noteKey) {
   return input.replace(new RegExp(mdurl.encode(path.sep), 'g'), path.sep).replace(new RegExp(STORAGE_FOLDER_PLACEHOLDER + '(' + escapeStringRegexp(path.sep) + noteKey + ')?', 'g'), DESTINATION_FOLDER)
 }
 
+/**
+ * @description replace all :storage references with given destination folder.
+ * @param input Input in which the references should be deleted
+ * @param noteKey Key of the current note
+ * @param destinationFolder Destination folder of the attachements
+ * @returns {String} Input without the references
+ */
+function replaceStorageReferences (input, noteKey, destinationFolder) {
```

Comment:
Would it be possible that you add some tests for this method? e.g. for a note with one attachment and one test with multiple attachments?